source/_docs/migrate-troubleshooting.md

```diff
@@ -15,6 +15,14 @@ Next, check [log files](https://pantheon.io/docs/logs/) to help identify and fix
 ## One Application per Site
 Each site supports a single Drupal or WordPress application. Placing a WordPress application inside a Drupal site, for example, is unsupported. Drupal multisite is also not supported. If your existing Drupal site is a multisite installation, see [Extracting Sites from a Drupal Multisite](https://pantheon.io/docs/unwind-multisite/).
 
+## Multiple File Directories Found within the Import Archive
+The import will fail if Drupal's private files directory is not placed within the public directory, resulting in the following error message:
+
+![Multiple file directories found within the import archive](/source/docs/assets/images/multiple-files-dir-import-failure.png)
```

Comment:
Does it make sense to just have the error in plaintext? I want to make sure it's easily searchable. Also, there are several other unsupported scenarios that we'll want to add soon.